sdks/python/apache_beam/transforms/display.py

```diff
@@ -137,9 +137,15 @@ def create_payload(dd):
       except ValueError:
         # Skip if the display data is invalid.
```

Comment:
Consider printing a warning here if it makes sense.

Reply:
I'd rather not at this point given that DisplayData is only supported by Dataflow yet and existing transforms may end up producing warnings for all runners.

Reply:
sgtm.